e2e/cmd/adminpolicy/e2e_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2024 Tigera, Inc. All rights reserved.
```

Comment:
I copied this file from here: https://github.com/kubernetes-sigs/network-policy-api/blob/main/conformance/conformance_test.go
I believe I should use their copy right clause, right?

Reply:
Yes, always keep external copyright notices intact.  We can also add ours as and when we make changes (but best to make changes in a second file where possible to make it easier to unpick what's ours and what's theirs).